bucket/windows-terminal.json

```
@@ -0,0 +1,37 @@
+{
+    "homepage": "https://github.com/microsoft/terminal",
+    "description": "The new Windows Terminal, and the original Windows console host - all in the same place!",
+    "version": "0.6.2951.0",
+    "license": "MIT",
+    "url": "https://github.com/microsoft/terminal/releases/download/v0.6.2951.0/Microsoft.WindowsTerminal_0.6.2951.0_8wekyb3d8bbwe.msixbundle#/dl.7z",
+    "hash": "1e55cd46043ad5613242aae3bceb528b81a181a8b388015ebe7767000102d06a",
+    "architecture": {
+        "64bit": {
+            "pre_install": "Get-ChildItem \"$dir\" -Exclude '*x64.msix' | Remove-Item -Force -Recurse"
+        },
+        "32bit": {
+            "pre_install": "Get-ChildItem \"$dir\" -Exclude '*x86.msix' | Remove-Item -Force -Recurse"
+        }
+    },
+    "installer": {
+        "script": [
+            "$winVer = [Environment]::OSVersion.Version",
+            "if (($winver.Major -lt '10') -or ($winVer.Build -lt 18362)) { error 'At least Windows 10 18362 is required.'; break }",
+            "Get-ChildItem \"$dir\" '*.msix' | Select-Object -ExpandProperty Fullname | Expand-7zipArchive -DestinationPath \"$dir\" -Removal"
+        ]
+    },
+    "bin": "WindowsTerminal.exe",
+    "shortcuts": [
+        [
+            "WindowsTerminal.exe",
+            "Windows Terminal (Preview)"
```

I would not enforce to follow this Microsoft store naming as there is no other version available right now.

```suggestion
            "Windows Terminal"
```

---

Is this the only change that's still blocking the manifest from being merged? @Bassetts

---

Guys, can we make this change accepted? New version is already out. Basically this manifest is outdated even before appearing in the bucket...

---

They have just released another version like five hours ago :) We falling behind very badly.